VisualPinball.Unity/VisualPinball.Unity.Editor/Import/VpxImportEngine.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.IO;
+using NLog;
+using UnityEditor;
+using UnityEngine;
+using VisualPinball.Unity.Import;
+using VisualPinball.Unity.Import.Job;
+using Logger = NLog.Logger;
+
+namespace VisualPinball.Unity.Editor.Import
+{
+	public static class VpxImportEngine

```

Comment:
Can we maybe merge `VpxImportEngine` and `VpxMenuImporter` and just call it `VpxImporter`? `VpxMenuImporter.ImportVpxEditorMemory` is obsolete now anyway, and otherwise there's much code in there.

Reply:
In my opinion the importer should be a dedicated class, it should have got nothing to do with the menu. That way one can use it everywhere. This was my use case, i. e. have 1 class that does the import, it's called in the import menu and from the import wizard window and wherever anyone prefers, e. g. from within a script that imports 5 tables just to test out how the importer works.

Reply:
Yeah, I don't like it how it's right now, we have `VpxImportEngine`, `VpxImportWizard`, `VpxMenuImporter`, `VpxImporter`, some are `MonoBehaviour`, some static classes, it's a bit messy.

It's not your fault though, and I'll merge this and create a refactor issue.